config/locales/en.yml

```diff
@@ -764,61 +764,61 @@ en:
           marketplace: "Marketplace"
           mobile_money_agent: "Mobile Money Agent"
           monastery: "Monastery"
-          money_transfer: "Money Transfer"
+          money_transfer: "Money Transfer Station"
           motorcycle_parking: "Motorcycle Parking"
           music_school: "Music School"
-          nightclub: "Night Club"
+          nightclub: "Nightclub"
           nursing_home: "Nursing Home"
-          parking: "Parking"
-          parking_entrance: "Parking Entrance"
+          parking: "Parking Lot"
+          parking_entrance: "Parking Garage Entrance"
           parking_space: "Parking Space"
           payment_terminal: "Payment Terminal"
           pharmacy: "Pharmacy"
           place_of_worship: "Place of Worship"
           police: "Police"
-          post_box: "Post Box"
+          post_box: "Mail Drop Box"
           post_office: "Post Office"
           prison: "Prison"
           pub: "Pub"
           public_bath: "Public Bath"
           public_bookcase: "Public Bookcase"
           public_building: "Public Building"
           ranger_station: "Ranger Station"
-          recycling: "Recycling Point"
+          recycling: "Recycling"
           restaurant: "Restaurant"
-          sanitary_dump_station: "Sanitary Dump Station"
+          sanitary_dump_station: "RV Toilet Disposal"
```

Comment:
I'm not too familiar with RVs, but it seems like this is may still called a dump station in American English, just not "sanitary" dump station. "RV dump station" seems commonly used.

Reply:
Thanks, I’m not very familiar with these facilities, so I just went with id-tagging-schema on this one. I’ll change it to “RV Dump Station”.